### ISSUE TITLE
facebook rebound library issue

### ISSUE THREAD

**Issue by jjhesk** [novoda/bintray-release#38]
I have found the issue when I am trying include facebook rebound library as stated in gradle with the current version 0.2.7
```gradle
apply plugin: 'com.android.library'


ext {
    PUBLISH_GROUP_ID = 'com.hkm'
    PUBLISH_ARTIFACT_ID = project.POM_ARTIFACT_ID
    PUBLISH_VERSION = project.VERSION_NAME
}
android {
    compileSdkVersion Integer.parseInt(project.ANDROID_BUILD_SDK_VERSION)
    buildToolsVersion project.ANDROID_BUILD_TOOLS_VERSION

    defaultConfig {
        minSdkVersion Integer.parseInt(project.ANDROID_BUILD_MIN_SDK_VERSION)
        targetSdkVersion Integer.parseInt(project.ANDROID_BUILD_TARGET_SDK_VERSION)
        versionName project.VERSION_NAME
        versionCode Integer.parseInt(project.VERSION_CODE)
    }

    lintOptions {
        abortOnError false
    }
}


dependencies {
    compile 'com.android.support:support-annotations:21.0.2'
    compile 'com.squareup.okhttp:okhttp-urlconnection:2.2.0'
    compile 'com.android.support:recyclerview-v7:21.0.2'
    compile 'com.facebook.rebound:rebound:0.3.7'
    compile 'de.hdodenhof:circleimageview:1.2.2'
    compile 'com.makeramen:roundedimageview:1.5.0'
    compile 'com.squareup.picasso:picasso:2.5.0'
}


apply from: 'bintray.gradle'

```

the gradle file:
```gradle
apply plugin: 'bintray-release'

publish {
    userOrg = project.POM_DEVELOPER_ID
    groupId = project.PUBLISH_GROUP_ID
    artifactId = project.POM_ARTIFACT_ID
    version = project.VERSION_NAME
    uploadName = project.POM_ARTIFACT_ID
    description = project.POM_DESCRIPTION
    website = project.POM_URL
    autoPublish = true
    dryRun = false
    licences=[project.POM_LICENCE_NAME]
}

if (hasProperty('POM_DEVELOPER_ID')) {
    publish.bintrayUser = project.POM_DEVELOPER_ID
}

if (hasProperty('bintray.apikey')) {
    publish.bintrayKey = property('bintray.apikey')
}

```
and the result is shown as below:

```

10:15:00 AM: Executing external task 'bintrayUpload'...
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:22: error: package com.facebook.rebound does not exist
import com.facebook.rebound.Spring;
                           ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:23: error: package com.facebook.rebound does not exist
import com.facebook.rebound.SpringListener;
                           ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:24: error: package com.facebook.rebound does not exist
import com.facebook.rebound.SpringSystem;
                           ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:25: error: package com.facebook.rebound does not exist
import com.facebook.rebound.SpringUtil;
                           ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:26: error: cannot find symbol
import hkm.ui.expendablefragment.R;
                                ^
  symbol:   class R
  location: package hkm.ui.expendablefragment
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:33: error: cannot find symbol
import static hkm.ui.expendablefragment.R.styleable;
                                       ^
  symbol:   class R
  location: package hkm.ui.expendablefragment
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:33: error: static import only from classes and interfaces
import static hkm.ui.expendablefragment.R.styleable;
^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:38: error: cannot find symbol
public class CatelogView<T extends Fragment & FragmentClickable> extends LinearLayout implements View.OnClickListener, SpringListener, FragmentInterface {
                                                                                                                       ^
  symbol: class SpringListener
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:41: error: cannot find symbol
    private static SpringSystem springSystem = SpringSystem.create();
                   ^
  symbol:   class SpringSystem
  location: class CatelogView<T>
  where T is a type-variable:
    T extends Fragment,FragmentClickable declared in class CatelogView
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:8: error: cannot find symbol
import hkm.ui.expendablefragment.R;
                                ^
  symbol:   class R
  location: package hkm.ui.expendablefragment
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:62: error: cannot find symbol
    private Spring spring;
            ^
  symbol:   class Spring
  location: class CatelogView<T>
  where T is a type-variable:
    T extends Fragment,FragmentClickable declared in class CatelogView
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:334: error: cannot find symbol
    public void onSpringAtRest(Spring spring) {
                               ^
  symbol:   class Spring
  location: class CatelogView<T>
  where T is a type-variable:
    T extends Fragment,FragmentClickable declared in class CatelogView
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:344: error: cannot find symbol
    public void onSpringActivate(Spring spring) {
                                 ^
  symbol:   class Spring
  location: class CatelogView<T>
  where T is a type-variable:
    T extends Fragment,FragmentClickable declared in class CatelogView
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:349: error: cannot find symbol
    public void onSpringEndStateChange(Spring spring) {
                                       ^
  symbol:   class Spring
  location: class CatelogView<T>
  where T is a type-variable:
    T extends Fragment,FragmentClickable declared in class CatelogView
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:358: error: cannot find symbol
    public void onSpringUpdate(Spring S) {
                               ^
  symbol:   class Spring
  location: class CatelogView<T>
  where T is a type-variable:
    T extends Fragment,FragmentClickable declared in class CatelogView
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/bindingholder/basicViewHolder.java:6: error: cannot find symbol
import hkm.ui.expendablefragment.R;
                                ^
  symbol:   class R
  location: package hkm.ui.expendablefragment
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/bindingholder/SlickHolder.java:6: error: cannot find symbol
import hkm.ui.expendablefragment.R;
                                ^
  symbol:   class R
  location: package hkm.ui.expendablefragment
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/header/FeatureImage.java:12: error: cannot find symbol
import hkm.ui.expendablefragment.R;
                                ^
  symbol:   class R
  location: package hkm.ui.expendablefragment
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/header/FeatureImage.java:13: error: package com.squareup.picasso does not exist
import com.squareup.picasso.Picasso;
                           ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/list/SlickListAdapter.java:5: error: cannot find symbol
import hkm.ui.expendablefragment.R;
                                ^
  symbol:   class R
  location: package hkm.ui.expendablefragment
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/list/SlickListAdapter.java:10: error: package com.squareup.picasso does not exist
import com.squareup.picasso.Picasso;
                           ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/list/SlickListAdapter.java:18: error: cannot find symbol
    final Picasso theloadingimagepicasso = Picasso.with(getContext());
          ^
  symbol:   class Picasso
  location: class SlickListAdapter
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:322: warning: no description for @param
     * @param v
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:331: warning: no description for @param
     * @param spring
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:341: warning: no description for @param
     * @param spring
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogView.java:355: warning: no description for @param
     * @param S
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:164: warning: no description for @param
     * @param styleId
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:165: warning: no description for @return
     * @return
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:175: warning: no description for @param
     * @param resId
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:176: warning: no description for @return
     * @return
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:186: warning: no description for @return
     * @return
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:195: warning: no description for @param
     * @param str
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:196: warning: no description for @return
     * @return
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:210: warning: no description for @return
     * @return
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/CatelogViewBuilder.java:223: warning: no description for @return
     * @return
       ^
/Users/hesk/github/expandableWidget/eexpandedview/src/main/java/hkm/ui/expendablefragment/header/FeatureImage.java:16: error: unknown tag: teach
 * @teach you how to write parcel http://blog.callumtaylor.net/serialisationjava-android
   ^
:eexpandedview:androidJavadocs
1 error
35 warnings
:eexpandedview:androidJavadocs FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':eexpandedview:androidJavadocs'.
> Javadoc generation failed. Generated Javadoc options file (useful for troubleshooting): '/Users/hesk/github/expandableWidget/eexpandedview/build/tmp/androidJavadocs/javadoc.options'

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 4.757 secs
Process 'command '/Library/Java/JavaVirtualMachines/jdk1.8.0_31.jdk/Contents/Home/bin/javadoc'' finished with non-zero exit value 1
10:15:06 AM: External task execution finished 'bintrayUpload'.


```

**Comment by ouchadam** [comment]
seems like this is the same issue as #25 . As a quick fix you might be able to run `--continue` to force the script to continue or `-x androidJavaDocs` to skip the java docs task. 

Thanks for reporting! We'll look into it :+1:

**Comment by jjhesk** [comment]
with the ui configuration, do i just need to add 
``` --continue x androidJavaDocs ``` on the field with Script parameters in the Run/Debug Configurations window?

**Comment by ouchadam** [comment]
on the command line the usage would be `./gradlew bintrayUpload -x androidJavaDoc` or `./gradlew bintrayUpload --contine`

I'm not familiar with using the UI but script parameters should take `-x androidJavaDoc` or `-x eexpandedview:androidJavadocs`

**Comment by jjhesk** [comment]
i tried and that should be ```-x androidJavaDoc```

**Comment by ouchadam** [comment]
fixed in `0.2.10`
